README.md

```diff
@@ -52,6 +53,10 @@ jobs:
 
 ## Configuration
 
+### packageManager
```

Comment:
Unfortunately this header makes all the following paragraphs appear as if they were part of the packageManager docs.